client/driver/rkt.go

```diff
@@ -53,20 +55,27 @@ type RktDriverConfig struct {
 
 // rktHandle is returned from Start/Open as a handle to the PID
 type rktHandle struct {
-	proc        *os.Process
-	image       string
-	logger      *log.Logger
-	killTimeout time.Duration
-	waitCh      chan *cstructs.WaitResult
-	doneCh      chan struct{}
+	pluginClient *plugin.Client
+	userPid      int
```

Comment:
This won't be the userpid in the case of a rkt container. This would be the pid of the rkt tool, the user pid would be running inside the rkt container. Can we please rename the variable?